### ISSUE TITLE
how to cite secondary sources APA style

### ISSUE THREAD

**Issue by akramwahid** [NCSU-Libraries/citationbuilder#1]
Hi, can you please tell me whether this library supports citing secondary sources?

**Comment by erikncsu** [comment]
Yes, if you cite a source that uses another source, technically that is citing the secondary source. 

Did you have more of a specific use case?

**Comment by akramwahid** [comment]
You are correct, citing a source that uses another source is secondary source , however i could not find such option in your demo, for ex: 
Allport's diary (as cited in Nicholson, 2003).

Is it possible to achieve this in your demo

**Comment by erikncsu** [comment]
No, the Citation Builder does not offer help with what the Purdue Online Writing Lab calls "indirect sources." See the "Citing indirect sources" sections of the Purdue Online Writing Lab's guide to MLA and APA.

Purdue's MLA guide shows this as an example of an in-text citation:

    Ravitch argues that high schools are pressured to act as "social service centers, and they don't do that well" (qtd. in Weisman 259).

Our Citation Builder could help you properly construct a citation to the Weisman book for your works cited page.
